recipes/pantheon/builder.js

```diff
@@ -19,6 +19,9 @@ const overrideAppserver = options => {
   options.services.appserver.overrides.volumes.push(`${options.confDest}/prepend.php:/srv/includes/prepend.php`);
   // Add in our environment
   options.services.appserver.overrides.environment = utils.getPantheonEnvironment(options);
+
+  // Override the Solr8 service.
+  options.services.index.overrides.volumes.push(`${options.confDest}/jetty.xml:/opt/solr-8.11.2/server/etc/jetty.xml`);
```

Comment:
@reynoldsalec the comment says this will override the solr 8 service but i assume this actually overrides BOTH versions. Similar to other comment we probably want to make this conditional to decrease risk of CROSS CONTAMINATION